packages/govern-console/src/components/Header/Header.tsx

```diff
@@ -47,7 +56,23 @@ function Header() {
           align-items: center;
         `}
       >
-        <img src={AragonSvg} width="36" alt="Grey Eagle Aragon logo" />
+        <button
+          type="button"
+          onClick={handleGoToHome}
+          css={`
+            background: transparent;
+            border: 0px;
+            &:hover {
+              cursor: pointer;
+            }
+            &:active {
+              position: relative;
+              top: 1px;
+            }
+          `}
+        >
+          <img src={AragonSvg} width="36" alt="Grey Eagle Aragon logo" />
```

Comment:
We should use an empty alt here, since we have the title right after it (“Govern Console”). When a logo is being used in a header, it is generally recommended to use either an empty alt (when the textual version is present) or the name of the site / app directly, rather than describing the image itself. This is to avoid screen readers repeating “Grey Eagle Aragon logo − Govern Console” every time the page loads :smile: 

Now since we are adding a button here, we need a way to describe it. We could use a `title` to describe the action, but should we move the `h1` inside the button instead? I think it would make sense to make them both clickable.

Reply:
I had no idea about this for screen readers! Will do. :) Do you have any material regarding this that I could read? And yes, let's actually move it inside the button. Looking back, it feels weird that you can click the logo but not the title. 😄

Reply:
> Do you have any material regarding this that I could read?

Of course!

The WAI tutorial about images describes a case similar to this one: https://www.w3.org/WAI/tutorials/images/functional/#logo-image-within-link-text

Except in our case, the text doesn’t describe the actual action… but a click on a logo is generally expected to have the “go to home” behavior, so I usually omit it. This is also [what the WebAIM website recommends](https://webaim.org/techniques/alttext/#logos):

> Developers will often identify the image as being to the home page (alt="Acme Company home page), though if the image is consistently at the beginning of the page and the alternative text is appropriate, this additional information should not be needed.